docs/bucket/notifications/README.md

```diff
@@ -894,191 +1038,174 @@ The Minio server configuration file is stored on the backend in json format. Upd
+Use the following command to check if the event notification was successful:
+
+```sh
+mc events list myminio/images
 ```
 
-Thumbnailer starts running at ``http://localhost:3000/``. Next, configure the Minio server to send notifications to this URL (as mentioned in step 1) and use ``mc`` to set up bucket notifications (as mentioned in step 2). Then upload a JPEG image to Minio server by
+A response similar to this one should be displayed:
 
 ```
-mc cp ~/images.jpg myminio/images
-.../images.jpg:  8.31 KB / 8.31 KB ┃▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓┃ 100.00% 59.42 KB/s 0s
+arn:minio:sqs::1:webhook   s3:ObjectCreated:*   Filter: suffix=".jpg"
 ```
-Wait a few moments, then check the bucket’s contents with mc ls — you will see a thumbnail appear.
 
-```
-mc ls myminio/images-thumbnail
-[2017-02-08 11:39:40 IST]   992B images-thumbnail.jpg
-```
+### 10.3 Test with Thumbnailer
 
-<a name="NSQ"></a>
-## Publish Minio events to NSQ
+[Thumbnailer](https://github.com/minio/thumbnailer) can be used to listen for Minio Server notifications when a new JPEG file is uploaded using HTTP PUT. Thumbnailer uploads the thumbnail of a new image to Minio Server when a notification occurs.
 
-Install an NSQ Daemon from [here](https://nsq.io/). Or use the following Docker
-command for starting an nsq daemon:
+Use the following commands to download and install Thumbnailer:
 
-```
-docker run --rm -p 4150-4151:4150-4151 nsqio/nsq /nsqd
+```sh
+git clone https://github.com/minio/thumbnailer/
+npm install
 ```
 
-### Step 1: Add NSQ endpoint to Minio
+Open the Thumbnailer configuration file located in ``config/webhook.json`` and add the configuration for Minio Server.
 
-The Minio server configuration file is stored on the backend in json format. The NSQ configuration is located in the `nsq` key under the `notify` top-level key. Create a configuration key-value pair here for your NSQ instance. The key is a name for your NSQ endpoint, and the value is a collection of key-value parameters.
+Use the following command to run Thumbnailer at `http://localhost:3000/`:
 
-An example configuration for NSQ is shown below:
-
-```json
-"nsq": {
-    "1": {
-        "enable": true,
-        "nsqdAddress": "127.0.0.1:4150",
-        "topic": "minio",
-        "tls": {
-            "enable": false,       
-            "skipVerify": true
-        }
-    }
-}
-```
-To update the configuration, use `mc admin config get` command to get the current configuration file for the minio deployment in json format, and save it locally.
-```sh
-$ mc admin config get myminio/ > /tmp/myconfig
-```
-After updating the NSQ configuration in /tmp/myconfig , use `mc admin config set` command to update the configuration for the deployment.Restart the Minio server to put the changes into effect. The server will print a line like `SQS ARNs: arn:minio:sqs::1:nsq` at start-up if there were no errors.
 ```sh
-$ mc admin config set myminio < /tmp/myconfig
+NODE_ENV=webhook node thumbnail-webhook.js
```

Comment:
Is this supposed to be an `export`?

Reply:
@eco-minio "NODE_ENV=webhook node thumbnail-webhook" was the command provided to start the server but i couldn't get thumbnailer to work - just says "Error: Cannot find module '.../Minio/gluegun_clone/gluegun/thumbnail'" when i run that command. Can you please suggest a correction or advise in this case?